### DISCUSSION TITLE
Remix example with Auth flow and realtime data

### DISCUSSION THREAD

**Discussion by jonshipman** [pocketbase/pocketbase#4118] [Show and tell]
https://github.com/jonshipman/remix-run-examples/commit/73dba077bb3dfbabf70b41ee7908972eee99f22e

It's currently waiting merging into remix/run-examples, but I thought I'd share here in case it helps anyone. Doesn't go too outside the box, Remix does not currently support middleware so on every page you'll need to call `const pb = createPocketbase(request)` in every `loader` function, but it works. If you're working in a larger project, you can roll your own express server according to the docs, but I felt it would be outside this example.

- auto downloads pocketbase 0.20.5 by way of scripts/install.mjs
- runs remix dev and pocketbase serve with `concurrently` (uses a scripts/pocketbase.mjs wrapper to account for os differences)
- has pb_migrations for creating a default admin and app user
- cronAdd function in pb_hooks to autoincrement a `realtime_example` collection record that's visible once you login under http://localhost:3000/admin

**Comment by deadcoder0904** [comment]
thanks for this. will check out.
